internal/driver/rest/handler.go

```diff
@@ -226,7 +226,7 @@ func (h *Handler) HandleProjectDetail(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if modelProject.Status == _type.PROJECT_NEED_REVIEW {
+	if modelProject.Status == _type.PROJECT_NEED_REVIEW && r.Context().Value("isAdmin") == false {
```

Comment:
The code doesn't work for me. Another alternative is using this utils
 ``` 
!role.HasRole From Context(ctx, _type.ROLE_ADMIN) 
```

Reply:
oke